meta/parse.pl

```diff
@@ -1640,6 +1644,10 @@ sub ProcessDefaultValue
     {
         WriteSource "$val = { .mac = { 0, 0, 0, 0, 0, 0 } };";
     }
+    elsif ($default =~ /^0\,0$/ and $type =~ /^(sai_vlan_stacking_vid_t)/)
```

Comment:
not need for \\, since comma is not regex special

Reply:
Corrected